rootfs/bin/boot

```diff
@@ -83,7 +83,7 @@ else
 fi
 
 # run an idempotent database migration
-sudo -E -u deis ./manage.py syncdb --migrate --noinput
+sudo -E -u deis ./manage.py migrate --noinput
```

Comment:
https://docs.djangoproject.com/en/1.7/releases/1.7/#schema-migrations

`syncdb` is an alias for `migrate` but `--migrate` got removed so I figured I'd just get us up to scratch off the bat